atlasdb-autobatch/src/test/java/com/palantir/atlasdb/autobatch/TracingEventHandlerTest.java

```diff
@@ -25,6 +25,7 @@
 import org.junit.runner.RunWith;
 import org.mockito.junit.MockitoJUnitRunner;
 
+/* TODO(boyoruk): Do not forget to upgrade this to JUnit5. */
```

Comment:
Adding explicit comment to remember to move this later. Add corresopnding Asana task.